tests/ui-tests/pages/login-page/login-page.ts

```diff
@@ -28,7 +28,7 @@ export class Authentication extends Master {
             await this.loginButton.click();
             await this.page.waitForURL(`**/${this.username}`, {waitUntil: 'domcontentloaded'});
 
-            if (this.page.url() === `https://www.themoviedb.org/u/${this.username}`) {
+            if(await this.userInfoButtonToBeVisible){
```

Comment:
I assuem this function is not called actually. Issues like this could be eliminated using static code analysis tooling.